src/fidesops/task/graph_task.py

```diff
@@ -168,6 +177,27 @@ def can_write_data(self) -> bool:
         connection_config: ConnectionConfig = self.connector.configuration
         return connection_config.access == AccessLevel.write
 
+    def _combine_seed_data(
+        self,
+        *data: List[Row],
+        grouped_data: Dict[str, Any],
+        dependent_field_mappings: COLLECTION_FIELD_PATH_MAP,
+    ) -> Dict[str, Any]:
+        """Combine the seed data with the other dependent inputs. This is used when seed data into a collection requires
```

Comment:
Should read `...when seeding data...`

Reply:
hm I meant it as a noun, maybe adding `the` will help? This is used when the seed data in a collection requires inputs from another collection to generate subsequent queries.